client/src/views/TournamentView.tsx

```diff
@@ -253,14 +265,14 @@ const TournamentView = () => {
                 >
                   {round.map((fight) => {
                     const fighters = JSON.parse(fight.fighters) as Fighter[];
-
+                    const brute1 : Fighter | undefined = fighters.find((fighter) => !fighter.master && fighter.type === 'brute' && fighter.team === 'L');
+                    const brute2 : Fighter | undefined = fighters.find((fighter) => !fighter.master && fighter.type === 'brute' && fighter.team === 'R');
```

Comment:
It's always better to not force any typings, as they are already set.

```suggestion
                    const brute1 = fighters.find((fighter) => !fighter.master && fighter.type === 'brute' && fighter.team === 'L');
                    const brute2 = fighters.find((fighter) => !fighter.master && fighter.type === 'brute' && fighter.team === 'R');
```